sync/class.jetpack-sync-functions.php

```diff
@@ -203,26 +210,26 @@ public static function get_protocol_normalized_url( $callable, $new_value ) {
 		$option_key = self::HTTPS_CHECK_OPTION_PREFIX . $callable;
 
 		$parsed_url = wp_parse_url( $new_value );
-		if ( ! $parsed_url ) {
+		if ( empty ( $parsed_url['scheme'] ) ) {
 			return $new_value;
 		}
 
-		$scheme = $parsed_url['scheme'];
-		$scheme_history = get_option( $option_key, array() );
+		$scheme           = $parsed_url['scheme'];
+		$scheme_history   = get_option( $option_key, array() );
 		$scheme_history[] = $scheme;
 
 		// Limit length to self::HTTPS_CHECK_HISTORY
-		$scheme_history = array_slice( $scheme_history, ( self::HTTPS_CHECK_HISTORY * -1 ) );
+		$scheme_history = array_slice( $scheme_history, ( self::HTTPS_CHECK_HISTORY * - 1 ) );
 
 		update_option( $option_key, $scheme_history );
 
-		$forced_scheme =  in_array( 'https', $scheme_history ) ? 'https' : 'http';
+		$forced_scheme = in_array( 'https', $scheme_history ) ? 'https' : 'http';
 
 		return set_url_scheme( $new_value, $forced_scheme );
 	}
 
 	public static function get_raw_url( $option_name ) {
-		$value = null;
+		$value    = null;
```

Comment:
Please avoid unnecessary changes like this. It's OK to remove spaces that shouldn't be there in related code, but these edits make the PR harder to review. Thanks!